tests/base/model_train_steps.py

```diff
@@ -21,12 +21,21 @@ def training_step(self, batch, batch_idx, optimizer_idx=None):
 
         # calculate loss
         loss_val = self.loss(y, y_hat)
+        loss_scalar = loss_val.item()
 
         # alternate possible outputs to test
+        if batch_idx % 2 == 0:
+            output = OrderedDict({
+                'loss': loss_val,
+                'progress_bar': {'some_val': loss_val * loss_val},
```

Comment:
I think this modulo 2 was recently changed on master. same below.

Reply:
Yup. But I wanted it to alternate between tensors and python scalars. Is there a better approach?

Reply:
If it's intentional, ok, but I think it got removed with the reason being that it can be harder to debug tests if the output format changes from iteration to iteration. I'm not trying to be an expert here xD so maybe better ask @Borda

Reply:
@Borda Do you think there's another way to address this?